<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelTime">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>25:25:61</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDate">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>32.13.2018</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWigdet">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab { height: 40px; width: 150px; }
/*QTabBar::tab:text
{
    border-top-color: #1D2A32;
    border-color: #40494E;
    color: white;
}*/
QTabBar::tab
{
    color: white;
    background-color: black;
    border-color: grey;
    border-bottom-color: black; /* same as pane color */
}
QTabBar::tab:selected
{ 
    background-color: rgb(40, 40, 40);
}</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tabEg">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <attribute name="title">
      <string>Temp EG</string>
     </attribute>
     <widget class="QLabel" name="labelTab1Temp1">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 48pt &quot;.SF NS Text&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>99.9째C</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelTab1Temp2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 48pt &quot;.SF NS Text&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>99.9째C</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelTab1Temp3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 48pt &quot;.SF NS Text&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>99.9째C</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelTab1Temp4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>160</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 48pt &quot;.SF NS Text&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>99.9째C</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelTab1Temp1Label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Wohnzimmer</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTab1Temp2Label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Arbeitszimmer</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTab1Temp3Label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Bad</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTab1Temp4Label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>140</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Terrasse</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabDg">
     <attribute name="title">
      <string>Temp DG</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabGarage">
     <attribute name="title">
      <string>Garage</string>
     </attribute>
     <widget class="QPushButton" name="pushButtonTor">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>240</width>
        <height>60</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
color: rgb(255, 255, 255);
background-color: rgb(0, 0,0);
border-color: rgb(40, 40, 40);
border-width: 3px;        
border-style: solid;
border-radius: 4px;
}
QPushButton:pressed
{
background-color: rgb(40,40,40)
}
</string>
      </property>
      <property name="text">
       <string>Tor</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTorstatus">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Tor ist offen!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>290</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ups ...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>doorOpenClose()</slot>
 </slots>
</ui>
